app/renderer/js/spellchecker.js

```diff
@@ -26,13 +32,12 @@ class SetupSpellChecker {
 		}
 	}
 
-	enableContextMenu() {
+	enableContextMenu(input) {
 		if (this.SpellCheckHandler) {
-			this.SpellCheckHandler.attachToInput();
-
-			const userLanguage = ConfigUtil.getConfigItem('spellcheckerLanguage');
-
-			this.SpellCheckHandler.switchLanguage(userLanguage);
+			setTimeout(() => this.SpellCheckHandler.attachToInput(), 1000);
+			const langname = lngDetector.detect(input, 1);
+			const langcode = ISO6391.getCode(langname);
+			this.SpellCheckHandler.switchLanguage(langcode);
```

Comment:
I'm guessing that this is meant to detect the language the user is typing and then change the spellchecker's language to that, but I don't see why the additional modules are required if `electron-spellchecker` is enough for that. 

(that's what I think this PR is meant for, please correct me if I'm wrong).